torch/csrc/jit/codegen/cuda/python_frontend/fusion_record.h

```diff
@@ -1510,6 +1511,95 @@ struct BatchNormOpRecord : RecordFunctor {
   bool channels_last_;
 };
 
+struct FullOpRecord : RecordFunctor {
```

Comment:
It would be good to craft this as a more generic `FactoryOpRecord`, where, at least, the skeleton is in place to also add `zeros` and `ones` with the same record.  For speed, I can understand not adding the rest for now, if that is non-trivial.

Reply:
Is there a benefit to having a `zeros` and `ones` record. All of the factory ops are defined in terms of `torch.full`.
https://github.com/pytorch/pytorch/blob/master/torch/_refs/__init__.py#L3743-L3844

Reply:
I did not realize that!  Thanks or pointing that out.  Maybe we would want to revisit this for other reasons but just `Full` seems good for now, under the circumstances.